openquake/hazardlib/gsim/munson_thurber_1997.py

```diff
@@ -106,13 +106,18 @@ class MunsonThurber1997Hawaii(MunsonThurber1997):
     Seismic-hazard maps for Hawaii. US Geological Survey; 2000.
     """
 
+    DEFINED_FOR_INTENSITY_MEASURE_TYPES = set([PGA, SA])
+
```

Comment:
You can also use `{PGA, SA}`.

Reply:
Is this syntax better? In other gsims I have checked as examples, I only see ([...])

Reply:
The old syntax is for Python 2, now we are using Python 3.